doc/user/content/cloud/cloud-deployments.md

```diff
@@ -25,6 +25,7 @@ The Details card shows you basic information about your deployment:
 - Name
 - Status
 - Hostname
+- Static IP
```

Comment:
```suggestion
- Static IP addresses
```

Reply:
Does it have to be plural? It is only 1 static IP per deployment.